### PR TITLE
Add a bookmarks plugin that supports names

### PR DESCRIPTION
**Pull request by toddyamakawa**
Here is a plugin to address #553

### CODE REVIEW

File: plugins/README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ Plugins are installed to `${XDG_CONFIG_HOME:-$HOME/.config}/nnn/plugins`.
 | --- | --- | --- | --- |
 | autojump | Navigate to dir/path | sh | autojump |
 | boom | Play random music from dir | sh | [moc](http://moc.daper.net/) |
+| bookmarks | Use named bookmarks managed with symlinks | sh | _see in-file docs_ |
```
Comment (leovilok) [inline plugins/README.md:31]: The only dependency listed in the script is `fzf`, so you could just write:
```suggestion
| bookmarks | Use named bookmarks managed with symlinks | sh | fzf |
```

File: plugins/bookmarks [hunk 2]
```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env sh
+
+# #############################################################################
```
Comment (jarun) [inline plugins/bookmarks:3]: Please remove the top and bottom ####. Keep keep uniform with other plugins.

File: plugins/bookmarks [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+# Description: Use named bookmarks using symlinks
+#
+# Shell: POSIX compliant
+# Author: Todd Yamakawa
+#
+# Dependencies: fzf
+#
+# How to use:
```
Comment (jarun) [inline plugins/bookmarks:11]: Usage:

File: plugins/bookmarks [hunk 4]
```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env sh
+
+# #############################################################################
+# Description: Use named bookmarks using symlinks
+#
+# Shell: POSIX compliant
```
Comment (jarun) [inline plugins/bookmarks:6]: Shell and Author at the end. Please remain uniform with other plugins.

File: plugins/bookmarks [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+#
+# Bonus tip: Add `$BOOKMARKS_DIR` to your `$CDPATH`
+#
+# TODO:
+#   1. Remove `fzf` dependency
+#
+# Shell: POSIX compliant
+# Author: Todd Yamakawa
+
+BOOKMARKS_DIR="$HOME/.nnn_bookmarks"
```
Comment (jarun) [inline plugins/bookmarks:27]: Please use `"${XDG_CACHE_HOME:-$HOME/.cache}/nnn/`

See plugin gutenread.

File: plugins/bookmarks [hunk 6]
```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env sh
+
+# Description: Use named bookmarks using symlinks
+#
+# Dependencies: fzf
```
Comment (jarun) [inline plugins/bookmarks:5]: Please change to:

Requires: fzf

Comment (jarun) [reply plugins/bookmarks:5]: Ignore it. Dependencies is better.

File: plugins/README.md [hunk 7]
```diff
@@ -28,6 +28,7 @@ Plugins are installed to `${XDG_CONFIG_HOME:-$HOME/.config}/nnn/plugins`.
 | --- | --- | --- | --- |
 | autojump | Navigate to dir/path | sh | autojump |
 | boom | Play random music from dir | sh | [moc](http://moc.daper.net/) |
+| bookmarks | Use named bookmarks managed with symlinks | sh | fzf |
```
Comment (jarun) [inline plugins/README.md:31]: Should be above boom. The order is alphabetical.

**Comment by jarun** [comment]
Thank you!

**Pull request merged**